docs/data-ai/ai/alert.md

```diff
@@ -4,6 +4,121 @@ title: "Alert on inferences"
 weight: 60
 layout: "docs"
 type: "docs"
-no_list: true
-description: "TODO"
+description: "Use triggers to send email notifications when inferences are made."
 ---
+
+At this point, you should have already set up and tested [computer vision functionality](/data-ai/ai/run-inference/).
+On this page, you'll learn how to use triggers to send alerts in the form of email notifications when certain detections or classifications are made.
```

Comment:
you'll need to capture both email & webhooks. webhooks are almost certainly the more sought after part

Reply:
Possibly we want to put the webhook info into and include and put that on all of the multiple alert pages...

Reply:
you mean the webhook info from the helmet tutorial to /data-ai/data/advanced/alert-data/? IMO that seems like a lot of info to add/could overwhelm the page but I can add it if you think necessary

Reply:
I ended up doing it like this here: https://docs-test.viam.dev/3723/manage/troubleshoot/alert/#webhooks just so that we also cover webhooks. The emails are nice but I think more people will need webhooks than emails so we need to mention them and provide the info for it. I think maybe we need to have a webhook reference page or something that we can then link to from the three pages. Or use this as an include and put it in all the places where this comes up

Reply:
made it into an include